test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/TestDynamicRegenerateHolderClasses.java

```diff
@@ -39,7 +39,6 @@
 public class TestDynamicRegenerateHolderClasses extends DynamicArchiveTestBase {
     static String CHECK_MESSAGES[] = {"java.lang.invoke.Invokers$Holder source: shared objects file (top)",
                                       "java.lang.invoke.DirectMethodHandle$Holder source: shared objects file (top)",
-                                      "java.lang.invoke.DelegatingMethodHandle$Holder source: shared objects file (top)",
```

Comment:
I don't understand why this change is needed. Could you please explain it?

Reply:
With the simplification of `MethodHandles.constant`, it seems invoking `MethodHandles.constant` no longer need to initialize `DelegatingMethodHandle` because there is no longer need to rebind some MethodHandle; anyways we are loading fewer classes now.